ci/jenkins/mellanox/scripts/start_ci.sh

```diff
@@ -94,9 +94,16 @@ EOF
     if [[ -z "${ANTREA_CNI_PR}${ANTREA_CNI_BRANCH}" ]];then
         change_image_name $ANTREA_CNI_HARBOR_IMAGE antrea/antrea-ubuntu
     fi
+
+    bash $WORKSPACE/antrea-cni/hack/generate-manifest.sh --dev-image "antrea/antrea-ubuntu" > $ARTIFACTS/antrea.yml
+    let status=status+$?
+    if [ "$status" != 0 ]; then
+        echo "ERROR: Failed to generate antrea manifest!"
+        return $status
+    fi
     
     if [[ -z "$(grep hw-offload $WORKSPACE/antrea-cni/build/yamls/antrea.yml)" ]];then
-        sed -i '/start_ovs/a\        - --hw-offload' $WORKSPACE/antrea-cni/build/yamls/antrea.yml
+        sed -i '/start_ovs/a\        - --hw-offload' $ARTIFACTS/antrea.yml
     fi
```

Comment:
this should be done through kustomize / generate-manifest.sh, by adding an `--hw-offload` option (similar to how we support Kind / the OVS netdev datapath)

Reply:
Done, added a commit to support hw-offload, can you have a look?